### ISSUE TITLE
[requirements] Security vulnerability in the dependencies

### ISSUE THREAD

**Issue by katiayn** [PythonBalkan/pyconbalkan#193]
Plz, check the security vulnerability alert [here](https://github.com/PythonBalkan/pyconbalkan/network/alert/requirements.txt/urllib3/open) for more info. 🙂

**Comment by virogenesis** [comment]
404

**Comment by katiayn** [comment]
Sorry, maybe you don't have access to it. This is the message related to `urllib3`:
```
urllib3
1 urllib3 vulnerability found in requirements.txt 12 days ago
Remediation
Upgrade urllib3 to version 1.24.2 or later. For example:

urllib3>=1.24.2
Always verify the validity and compatibility of suggestions with your codebase.

Details
CVE-2019-11324 More information
high severity
Vulnerable versions: < 1.24.2
Patched version: 1.24.2
The urllib3 library before 1.24.2 for Python mishandles certain cases where the desired set of CA certificates is different from the OS store of CA certificates, which results in SSL connections succeeding in situations where a verification failure is the correct outcome. This is related to use of the ssl_context, ca_certs, or ca_certs_dir argument.
```

**Comment by ulgens** [comment]
I tried to update `requirements.txt` file with `pip-compile` but somehow it gives exact same result with current requirements file and nothing changes. Does anyone know why it acts like this?
